scm-record/src/types.rs

```diff
@@ -288,6 +288,19 @@ impl File<'_> {
         (acc_selected, acc_unselected)
     }
 
+    /// Set the selection of all sections in this file.
+    pub fn set_toggled(&mut self, toggled: bool) {
```

Comment:
What do you think about calling it `set_selected()`? I find "toggled" (the adjective) a bit unclear. I think of "toggle" (the verb) as switching something between two states, but it's not obvious what to me what "set toggled" means. `set_selected()` would also better match the function's description.

Reply:
Updated to `checked`. I think a "selection" is an item or subset of items which can then be checked or toggled.